txpipe/covariance.py

```diff
@@ -667,6 +669,17 @@ def run(self):
             cl_sacc.save_fits(output_filename, overwrite=True)
             print("Saved power spectra with its Gaussian covariance")
 
+    def get_tracer_noise_from_sacc(self, cl_sacc):
+        # This could be done inside TJPCov:
+        # https://github.com/LSSTDESC/TJPCov/issues/31
+
+        tracer_noise = {}
+        for trn, tr in cl_sacc.tracers.items():
+            if 'nl_cp' in tr.metadata:
```

Comment:
Could you give it a more descriptive name than nl_cp, so newcomers can tell what it means?

This mainly applies to the name it has in the data file, but also useful in some places in the code itself as well.  e.g. `n_ell` and `n_ell_coupled` maybe?

Reply:
Done